src/components/farm/detail/ProductionForecast.tsx

```
@@ -0,0 +1,67 @@
+}
+
+export function ProductionForecast({ farm }: ProductionForecastProps) {
+  const t = useTranslations('farm.forecast');
+  const months = [t('months.january'), t('months.february'), t('months.march'), t('months.april'), t('months.may'), t('months.june'), t('months.july'), t('months.august'), t('months.september'), t('months.october'), t('months.november'), t('months.december')];
+  
+  const cropHarvestData : CropHarvestData = farm.activeCrops.reduce((acc, crop) => {
+    const mockMonths = ['Jul', 'Aug', 'Sep'];
+    const mockProgress = [58, 66, 75];
+    const index = farm.activeCrops.indexOf(crop) % 3;
+    return {
+      ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)